selfdrive/monitoring/driver_monitor.py

```diff
@@ -99,17 +102,22 @@ def __init__(self):
     self.cfactor = 1.
 
 class DriverStatus():
-  def __init__(self, rhd=False):
+  # pylint: disable=no-member
+  def __init__(self, rhd=False, **kwargs):
+    # init settings
+    self.__dict__.update(DRIVER_MONITOR_SETTINGS)
```

Comment:
this is pretty dirty. some cleaner options:
* move these constants into the DriverStatus class
* make the dict a class and acces with `MonitoringSettings.` instead of `self.`
* make the dict a class and DriverStatus a subclass

Reply:
kk let's use the setting class. Will fix after model work